.dockerignore

```diff
@@ -0,0 +1,7 @@
+# .dockerignore
+__pycache__
```

Comment:
We need to remove the commented lines i think
And I'm not sure, but it seems that docker container with your app will contain .git directory and something else unnecessary
result.png for example